fyodor/src/main/java/life/genny/fyodor/utils/FyodorUltra.java

```diff
@@ -139,7 +139,11 @@ else if (PRI_CREATED.equals(attributeCode)) {
 						ea.setValueDateTime(baseEntity.getCreated());
 					}
 					else {
-						ea = beaUtils.getEntityAttribute(baseEntity.getRealm(), baseEntity.getCode(), attributeCode, true, true);
+						try {
+							ea = beaUtils.getEntityAttribute(baseEntity.getRealm(), baseEntity.getCode(), attributeCode, true, true);
+						} catch (ItemNotFoundException e) {
+							log.trace(e.getMessage());
```

Comment:
could we warn here? Or add it to a set of missing eas in a row that we can then log once all the missing entity attributes for a row?

Reply:
lgtm otherwise

Reply:
It is such a generalised piece of code. If it were always the case that a missing EA was an error, then yes. But there will be many occasions where a field must display blank on a table for an entity.